public/locales/en/modules/media-requests-list.json

```diff
@@ -19,6 +19,37 @@
   },
   "tooltips": {
     "approve": "Approve requests",
-    "decline": "Decline requests"
+    "decline": "Decline requests",
+    "approving": "Approving Request..."
+  },
+  "mutation": {
+    "approving": "Approving",
+    "declining": "Declining",
+    "request": "request...",
+    "approved": "Request was approved!",
+    "declined": "Request was declined!"
+  },
+  "detail": {
+    "label": "Stats for nerds",
+    "id": "ID",
+    "device": "Device",
+    "video": {
+      "video":"Video",
+      "resolution": "Resolution",
+      "framerate": "Framerate",
+      "codec": "Video Codec"
+    },
+    "audio": {
+      "audio": "Audio",
+      "channels": "Audio Channels",
+      "codec": "Audio Codec"
+    },
+    "transcoding": {
+      "transcoding": "Transcoding",
+      "context": "Context",
+      "requested": "Hardware Encoding Requested",
+      "source": "Source Codec",
+      "target": "Target Codec"
+    }
```

Comment:
We made the conscious decision to not translate that. It's for nerds they should understand English, this way less things to translate and less space on page load (please revert this)

Reply:
We should provide those translations so the translators can select wheter or not they want to translate it. For the size it makes mostly no difference.